starting-a-journal/index.md

```diff
@@ -5,18 +5,18 @@ isBookIndex: true
 
 Here are the main steps in setting up your journal with OJS.  Some link to PKP School courses, which are free but require registration to take and to view.
 
-1. [Set up the technical infrastructure](http://pkpschool.sfu.ca/courses/becoming-an-editor/) - Module 4, Unit 4, including hosting, [domain name](https://www.wikihow.com/Pick-a-Good-Domain-Name-for-Your-Website), and server environment. Hosting is available from [PKP Publishing Services](https://pkpservices.sfu.ca/content/journal-hosting) or you can use your own host.
-2. [Protect your data](http://pkpschool.sfu.ca/courses/becoming-an-editor/)
+1. [Set up the technical infrastructure](https://pkpschool.sfu.ca/courses/becoming-an-editor) - See Module 2, Unit 4, including hosting, [domain name](https://www.wikihow.com/Pick-a-Good-Domain-Name-for-Your-Website), and server environment. Hosting is available from [PKP Publishing Services](https://pkpservices.sfu.ca/content/journal-hosting) or you can use your own host.
```

Comment:
@pmangahis Would it also make sense to link out to the admin guide here?

Reply:
Hi @kaitlinnewson, that would be a great idea